src/_assets/css/_bootstrap_config.scss

```diff
@@ -44,7 +44,7 @@ $alert-padding-x: bs-spacer(6);
 $border-radius: 0;
 $border-radius-lg: 0;
 $border-radius-sm: 0;
-$btn-font-weight: 400;
+// $btn-font-weight: 400; // Default is already normal (400). TODO: drop?
```

Comment:
This can be dropped, I had it as 400 because the site used to default to 300

Reply:
Done